### PR TITLE
Prevent CapsuleShape2D height from being less than zero.

### LINKED ISSUE #38514: Can't change axis of capsule collider
**Godot version:** 3.2.1

**OS/device including version:** Ubuntu 18.04

**Issue description:** If I use the control handles to modify the capsule so that it is wider than than it is tall, it turns invisible and the console says `Invalid polygon data, triangulation failed`.

This is fine:
image

And if I lower the handle on the top too much:
image

This case probably doesn't matter, but if both the handles coincide I get:
image

### PR DESCRIPTION
**Pull request by madmiraal**
Fixes #38514.

### CODE REVIEW

**Comment by MCrafterzz** [comment]
This is defiantly a better solution

**Review by MarcusElg — APPROVED** [review]

File: scene/resources/capsule_shape_2d.cpp [hunk 1]
```diff
@@ -72,6 +72,7 @@ real_t CapsuleShape2D::get_radius() const {
 void CapsuleShape2D::set_height(real_t p_height) {
 
 	height = p_height;
+	if (height < 0) height = 0;
```
Comment (akien-mga) [inline scene/resources/capsule_shape_2d.cpp:75]: Please don't put condition and statement on the same line.

Comment (madmiraal) [reply scene/resources/capsule_shape_2d.cpp]: Updated.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.2.

**Pull request merged**